src/components/OrganizationDetailPanel.tsx

```diff
@@ -8,15 +8,24 @@ import {
 	PanelTitleTags,
 	PanelTitleWrapper,
 } from './Panel';
+import { PanelGridItem } from './PanelGrid';
+import { ProposalListTable } from './ProposalListTable';
+import { DEFAULT_COLUMNS } from './ProposalListTablePanel';
```

Comment:
Trying to figure out my discomfort with this made me realize a larger design improvement we need, which predates you.

`DEFAULT_COLUMNS` is so named because we designed (but didn't implement yet) an interface for letting the user change which columns they want to see on the Dashboard (soon "Proposals") page. These are just the default/starting set.

But really the definition of the defaults should be within `ProposalListTable` itself, and `columns` should be an optional property.

So here's what I think we need to do instead:

1. Prior to this work, move `DEFAULT_COLUMNS` into `ProposalListTable.tsx`
2. Make `ProposalListTable`'s `columns` property optional, defaulting to `DEFAULT_COLUMNS`
   (I think it's _highly_ likely we'll want to tweak the columns here on the org panel after user testing, so this is a worthwhile configuration hook to have around.)
3. Remove any `column`-setting in `ProposalListTablePanel` and `OrganizationDetailPanel`

If you have the bandwidth to make that change, great, otherwise let me know and I can whip it out.